### PR TITLE
PrineMenuDone

### PR DESCRIPTION
**Pull request by ArtemNemtsov**

### CODE REVIEW

File: kitsune/menu/main.cpp [hunk 1]
```diff
@@ -27,11 +29,11 @@ void PopulateMenuFromFile(MenuList<std::string>& menu, const std::string& filena
     } 
 }
 
-void PrintEmptyString(int b = 1)
+void PrintEmptyString(int NumberOfSpaces = 1)
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: use such style: `numberOfSpaces`
Note that the first letter is in lower case.
It is better to rename this parameter. It is rather number of lines than number of spaces.

File: kitsune/menu/main.cpp [hunk 2]
```diff
@@ -53,9 +55,9 @@ void PrintMenu(const MenuList<std::string>& menu)
     
 */
 /* change code below */
-    std::cout << VERTICAL_BORDER << string(24, HORIZONTAL_BORDER) << VERTICAL_BORDER << '\n';
+    std::cout << VERTICAL_BORDER << string(WIDTH, HORIZONTAL_BORDER) << VERTICAL_BORDER << '\n';
     PrintEmptyString(1);
-    std::cout << VERTICAL_BORDER << string(10, ' ') << "MENU" << string(10, ' ') << VERTICAL_BORDER << '\n';
+    std::cout << VERTICAL_BORDER << string(WIDTH / 2 -2, ' ') << "MENU" << string(WIDTH / 2 -2, ' ') << VERTICAL_BORDER << '\n';
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: Please, pay attention to style. Put spaces around binary operators like `+`, `-`, `*`, `/`, `%`
Fix it everythere.

File: kitsune/menu/main.cpp [hunk 3]
```diff
@@ -65,32 +67,32 @@ void PrintMenu(const MenuList<std::string>& menu)
         {
             if (amountOfSymbol % 2 == 0)
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: it is sophisticated solution to mark activated items.
`if (i == activatedIndex) {/* some code here */ }` can change name of activated item from `"name"` to `"*name*"`. And further we can work with new item as usual with any other.

File: kitsune/menu/main.cpp [hunk 4]
```diff
@@ -65,32 +67,32 @@ void PrintMenu(const MenuList<std::string>& menu)
...
             {
-                std::cout << VERTICAL_BORDER << string(12 - amountOfSymbol / 2 , ' ')  << res[i];
-                std::cout << string(11 - amountOfSymbol / 2, ' ') << VERTICAL_BORDER << '\n';
+                std::cout << VERTICAL_BORDER << string(WIDTH /2 - amountOfSymbol / 2 , ' ')  << res[i];
+                std::cout << string(WIDTH /2 -1 - amountOfSymbol / 2, ' ') << VERTICAL_BORDER << '\n';
             }
             
         }
     }
-    PrintEmptyString (2);
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: It seems that function `PrintEmptyString` should be named as `PrintEmptyStrings`

File: kitsune/menu/main.cpp [hunk 5]
```diff
@@ -65,32 +67,32 @@ void PrintMenu(const MenuList<std::string>& menu)
...
-    std::cout << VERTICAL_BORDER << string(24, HORIZONTAL_BORDER) << VERTICAL_BORDER << '\n';
+    PrintEmptyString (EMPTY_STRING_AFTER_MENU);
```
Comment (eaniconer) [inline kitsune/menu/main.cpp]: Try to get rid of spaces between function name and open paranthesis.

Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: done. I found only this excess spaces  and deleted it

File: kitsune/menu/main.cpp (hunk 4 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: ok.Done

File: kitsune/menu/main.cpp (hunk 2 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: ок

File: kitsune/menu/main.cpp (hunk 1 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: done

File: kitsune/menu/main.cpp (hunk 3 above)
Comment (ArtemNemtsov) [reply kitsune/menu/main.cpp]: DOne

**Pull request merged**